Lib/idlelib/config_key.py

```diff
@@ -219,33 +224,34 @@ def TranslateKey(self, key, modifiers):
         return key
 
     def OK(self, event=None):
-        if self.advanced or self.KeysOK():  # doesn't check advanced string yet
-            self.result=self.keyString.get()
+        key = self.keyString.get()
+        key.strip()
+        if not key:
+            tkMessageBox.showerror(title=self.keyerror_title, parent=self,
+                                   message="No keys specified.")
```

Comment:
I changed line 6 way above from
import tkinter.messagebox as tkMessageBox  # to
from tkinter.messagebox import showerror
and deleted `tkMessageBox.` everywhere.
The same deletion will be needed in any other existing patches.